src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1802 @@
+        "r_stage3_stage2b": Parameter(
+            Types.REAL,
+            "probability per month of incident stage3 cervical cancer amongst people with stage2b",
+        ),
+        "r_stage4_stage3": Parameter(
+            Types.REAL,
+            "probability per month of incident stage4 cervical cancer amongst people with stage3",
+        ),
+        "rr_progress_cc_hiv": Parameter(
+            Types.REAL, "rate ratio for progressing through cin and cervical cancer stages if have unsuppressed hiv"
+        ),
+        "rr_hpv_vaccinated": Parameter(
+            Types.REAL,
+            "rate ratio for hpv if vaccinated - this is combined effect of probability the hpv is "
+            "vaccine-preventable and vaccine efficacy against vaccine-preventable hpv ",
+        ),
+        "rr_hpv_age50plus": Parameter(
+            Types.REAL,
+            "rate ratio for hpv if age 50 plus"
+        ),
+        "prob_cure_stage1": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 1 cervical cancer",
+        ),
+        "prob_cure_stage2a": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2a cervical cancer",
+        ),
+        "prob_cure_stage2b": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 2b cervical cancer",
+        ),
+        "prob_cure_stage3": Parameter(
+            Types.REAL,
+            "probability of cure if treated in stage 3 cervical cancer",
+        ),
+        "r_death_cervical_cancer": Parameter(
+            Types.REAL,
+            "probability per month of death from cervical cancer amongst people with stage 4 cervical cancer",
+        ),
+        "r_vaginal_bleeding_cc_stage1": Parameter(
+            Types.REAL, "rate of vaginal bleeding if have stage 1 cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage2a": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 2a cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage2b": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 2b cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage3": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 3 cervical cancer"
+        ),
+        "rr_vaginal_bleeding_cc_stage4": Parameter(
+            Types.REAL, "rate ratio for vaginal bleeding if have stage 4 cervical cancer"
+        ),
+        "prob_referral_biopsy_given_vaginal_bleeding": Parameter(
+            Types.REAL, "probability of being referred for a biopsy if presenting with vaginal bleeding"
+        ),
+        "sensitivity_of_biopsy_for_cervical_cancer": Parameter(
+            Types.REAL, "sensitivity of biopsy for diagnosis of cervical cancer"
+        ),
+        "sensitivity_of_xpert_for_hpv_cin_cc": Parameter(
+            Types.REAL, "sensitivity of xpert for presence of hpv, cin or cervical cancer"
```

Comment:
'cervical cancer' is being used intechangbly with 'stage4'... is that right? Can we stick to one name?

Reply:
stages 1-4 is cervical cancer